src/API/Trace/SpanContextInterface.php

```diff
@@ -12,14 +12,11 @@ interface SpanContextInterface
     public const TRACE_FLAG_SAMPLED = 0x01;
     public const TRACE_FLAG_DEFAULT = 0x00;
 
-    /** @todo Implement this in the API layer */
-    public static function createFromRemoteParent(string $traceId, string $spanId, int $traceFlags = self::TRACE_FLAG_DEFAULT, ?TraceStateInterface $traceState = null): SpanContextInterface;
-
     /** @todo Implement this in the API layer */
     public static function getInvalid(): SpanContextInterface;
 
     /** @todo Implement this in the API layer */
-    public static function create(string $traceId, string $spanId, int $traceFlags = self::TRACE_FLAG_DEFAULT, ?TraceStateInterface $traceState = null): SpanContextInterface;
+    public static function builder(string $traceId, string $spanId, bool $isRemote, int $traceFlags = self::TRACE_FLAG_DEFAULT, ?TraceStateInterface $traceState = null): SpanContextInterface;
```

Comment:
I don't think this is a builder, `create` made more sense.

Reply:
I would prefer `bool $remote = false` as last parameter.
Having an additional factory class feels unnecessary, this method is sufficient.

Implementation should check for invalid trace id / span id and return `::getInvalid()` for invalid values.